lms/djangoapps/certificates/models.py

```diff
@@ -233,6 +233,11 @@ class GeneratedCertificate(models.Model):
     # the course_modes app is loaded, resulting in a Django deprecation warning.
     from common.djangoapps.course_modes.models import CourseMode  # pylint: disable=reimported
 
+    # Normal object manager, which should only be used when ineligible
+    # certificates (i.e. new audit certs) should be included in the
+    # results. Django requires us to explicitly declare this.
+    objects = models.Manager()
```

Comment:
Order seems to matter here. Even though `objects` was being declared I couldn't access this attribute in a migration.

However, changing the order and running `makemigrations` seems to fix it up. I had access to the base object Manager, and the custom object Managers in a migration.